File: java/claude/p120.java
public class MemoryChunks {
    public static void main(String[] args) {
        // Allocate two chunks of memory each of size 10
        byte[] chunk1 = new byte[10];
        byte[] chunk2 = new byte[10];
        
        // Initialize first chunk with some data
        String data = "Hello";
        System.arraycopy(data.getBytes(), 0, chunk1, 0, data.length());
        
        // Copy the contents of the first chunk into the second chunk
        System.arraycopy(chunk1, 0, chunk2, 0, chunk1.length);
        
        // Print the contents of the second chunk
        System.out.println("Contents of second chunk: " + new String(chunk2).trim());
    }
}

File: python/claude/p120.py
# Allocate two chunks of memory each of size 10
chunk1 = bytearray(10)
chunk2 = bytearray(10)

# Initialize first chunk with some data
data = b"Hello"
chunk1[:len(data)] = data

# Copy the contents of the first chunk into the second chunk
chunk2[:] = chunk1[:]

# Print the contents of the second chunk
print("Contents of second chunk:", bytes(chunk2))
